frontend/public/locales/en/translation.json

```diff
@@ -1460,6 +1463,7 @@
   "GPU Accelerated Compute": "GPU Accelerated Compute",
   "Guest": "Guest",
   "H1 - 16 TB of HDD-based local storage": "H1 - 16 TB of HDD-based local storage",
+  "Has the words": "Has the words",
```

Comment:
I think we should call this "Fuzzy search" or something similar, as that will be more accurate. @berenss and @imjoyjean any thoughts on terminology there?

Reply:
I'm good with whatever decision Joy and Kevin make here!